recipes/melon/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.0-alpha.1":
```

Comment:
Do we have an idea on when proper `1.0.0` would be released?

Reply:
Actually, algorithms are well tested but I feel like the library lacks some features like serialization/visualization to be complete. I have been asked to submit a recipe and I think that making the library available on Conan-center could stimulate potential feedback and maybe contributions for designing these features.
Given that this is an important project for me, I'll continue working on it for sure :)

Reply:
Great to hear! Thanks for the extra insight